sound_play/package.xml

```diff
@@ -16,40 +16,38 @@
    <url type="repository">https://github.com/ros-drivers/audio_common</url>
    <url type="bugtracker">https://github.com/ros-drivers/audio_common/issues</url>
 
-   <buildtool_depend>catkin</buildtool_depend>
-   <buildtool_depend condition="$ROS_PYTHON_VERSION == 2">python-setuptools</buildtool_depend>
-   <buildtool_depend condition="$ROS_PYTHON_VERSION == 3">python3-setuptools</buildtool_depend>
+   <buildtool_depend>ament_cmake</buildtool_depend>
+   <buildtool_depend>ament_cmake_python</buildtool_depend>
+   <buildtool_depend>rosidl_default_generators</buildtool_depend>
+   <buildtool_depend>python3-setuptools</buildtool_depend>
 
-   <build_depend>roscpp</build_depend>
-   <build_depend>roslib</build_depend>
-   <build_depend>actionlib_msgs</build_depend>
-   <build_depend>actionlib</build_depend>
+   <build_depend>action_msgs</build_depend>
    <build_depend>audio_common_msgs</build_depend>
-   <build_depend>diagnostic_msgs</build_depend>
-   <build_depend>message_generation</build_depend>
+   <build_depend>boost</build_depend>
+   <build_depend>builtin_interfaces</build_depend>
 
-   <exec_depend>roscpp</exec_depend>
-   <exec_depend>roslib</exec_depend>
-   <exec_depend>actionlib_msgs</exec_depend>
+   <exec_depend>action_msgs</exec_depend>
    <exec_depend>audio_common_msgs</exec_depend>
-   <exec_depend>diagnostic_msgs</exec_depend>
+   <exec_depend>builtin_interfaces</exec_depend>
+   <exec_depend>festival</exec_depend>
+   <exec_depend>launch_xml</exec_depend>
+   <exec_depend>rclpy</exec_depend>
+   <exec_depend>rosidl_default_runtime</exec_depend>
 
-   <exec_depend condition="$ROS_PYTHON_VERSION == 2">python-gi</exec_depend>
-   <exec_depend condition="$ROS_PYTHON_VERSION == 3">python3-gi</exec_depend>
    <exec_depend>gstreamer1.0</exec_depend>
    <exec_depend>gstreamer1.0-alsa</exec_depend>
    <exec_depend>gstreamer1.0-plugins-base</exec_depend>
    <exec_depend>gstreamer1.0-plugins-ugly</exec_depend>
    <exec_depend>gstreamer1.0-plugins-good</exec_depend>
+   <exec_depend>python3-gi</exec_depend>
 
-   <exec_depend>rospy</exec_depend>
-   <exec_depend>festival</exec_depend>
    <exec_depend>flite</exec_depend>
-   <exec_depend>message_runtime</exec_depend>
    <exec_depend>resource_retriever</exec_depend>
 
+   <member_of_group>rosidl_interface_packages</member_of_group>
+
    <export>
-      <cpp cflags="-I${prefix}/include -I${prefix}/msg/cpp" />
-      <sound_play plugin="${prefix}/sound_play_plugin.yaml" />
+      <build_type>ament_cmake</build_type>
+      <sound_play plugin="${prefix}/launch/sound_play_plugin.yaml" />
```

Comment:
Is it ok to have `sound_play_plugin.yaml` file within the launch folder?

Reply:
I feel like even though having at root isn't great it is still better than the launch folder

Reply:
What about we use the resource folder for this kind of files?

Reply:
Mmm I wouldn't say it is a resource. I would put it within the sound_
play folder along the actual plugins or in the root of the package.

Reply:
Ok that makes sense. I would like to put them inside the folder where the actual python plugins are, it just seems a little bit more organized since those scripts are the only ones using the yaml files and not the "whole" package. Would that be ok?

Reply:
Yeah I agree